CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## 1.1.1 (unreleased)
 
+CHANGES:
+
+ * vault aws kms seal **breaking change**: user-configured regions will now be 
```

Comment:
Not sure how to do multi-line suggestions:
```
 * autoseal/aws: The user-configured regions on the AWSKMS seal stanza
   will now be preferred over regions set in the enclosing environment.
   This is a _breaking_ change.
```

Reply:
Thanks @calvn !

I could be wrong but I _think_ this is the way other things do it, like [this](https://github.com/hashicorp/vault/pull/6268/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR23) for instance, I think GH is just rendering it weird. I can make it a single line though if that seems better.

Reply:
My suggestion was mostly around the formatting on the sub-system mention (i.e. `autoseal/aws:` vs `vault aws kms seal **breaking change**:`). I ended up modifying the wording around the changelog to mention that it's a breaking change, but it ended up being a multi-line change and I've only used GH's suggestion feature of single-line suggestions.

Reply:
Oh I see what you mean! Yes, I will fix that right before I merge it so all the changelog updates are in one commit. I'll need to bump the version it's included in as well.